help/en/html/apps/PanelPro/SidebarAppLocalPart.shtml

```diff
@@ -4,13 +4,15 @@
 <dt><h3>PanelPro&trade;</h3></dt>
 <dd>
 <ul>
+    <li><a href="GettingStarted.shtml">Getting Started</a></li>
+    <li><a href="../LoadStoreWork.shtml">Loading and Storing</a></li>
     <li><a href="../../../package/jmri/jmrit/display/PanelEditor.shtml">Panel Editor</a></li>
     <li><a href="../../../package/jmri/jmrit/display/ControlPanelEditor.shtml">Control Panel Editor</a></li>
     <li><a href="../../../package/jmri/jmrit/display/LayoutEditor.shtml">Layout Editor</a></li>
     <li><a href="../../../package/jmri/jmrit/display/SwitchboardEditor.shtml">Switchboard Editor</a></li>
     <li><a href="FAQ.shtml">FAQ</a></li>
     <li><a href="/community/examples/Gallery.shtml">User's Examples</a></li>
-    <li><a href="/community/examples/Modules.shtml">Modular Layout Example</a></li>
+    <li><a href="/community/examples/Modules.shtml">Modular Layout Ex</a></li>
```

Comment:
For people that are not native English speakers, it's easier to understand `Modular Layout Example` than `Modular Layout Ex`.

Reply:
Done.